lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -22,6 +22,19 @@ class ListTools(ToolCommand):
     def __init__(self, context):
         super(ListTools, self).__init__(context)
 
+    @staticmethod
+    def print_results(toolinfo: dict, tool: str, with_option: bool):
+        for group in sorted(toolinfo.keys()):
+            for host in sorted(toolinfo[group].keys()):
+                tools = toolinfo[group][host]
+                if tools:
+                    if not with_option:
+                        s = ", ".join(sorted(tools.keys()))
+                    else:
+                        tools_with_options = [f"{t} {' '.join(tools[t])}" for t in sorted(tools.keys())]
+                        s = ", ".join( tools_with_options)
+                    print(f"group: {group}; host: {host}; tools: {s}")
```

Comment:
If I might suggest...
```
        for group, gval in sorted(toolinfo.items()):
            for host, tools in sorted(gval.items()):
                if tools:
                    if with_option:
                        tools_with_options = [f"{t} {' '.join(v)}" for t, v in sorted(tools.items())]
                        s = ", ".join(tools_with_options)
                    else:
                        s = ", ".join(sorted(tools.keys()))
                    print(f"group: {group}; host: {host}; tools: {s}")
```

Reply:
I did that except for the inversion of the tests: I like the longer body to go with the `else`.